src/platform/site-wide/announcements/config/index.js

```diff
@@ -2,10 +2,18 @@ import DashboardIntro from '../components/DashboardIntro';
 import Profile360Intro from '../components/Profile360Intro';
 import PersonalizationBanner from '../components/PersonalizationBanner';
 import ClaimIncreaseBanner from '../components/ClaimIncreaseBanner';
+import VAPlusVetsModal from '../components/VAPlusVetsModal';
 import isBrandConsolidationEnabled from '../../../brand-consolidation/feature-flag';
 
 const config = {
   announcements: [
+    {
+      name: 'brand-consolidation-va-plus-vets',
+      paths: /(.)/,
+      component: VAPlusVetsModal,
+      disabled: !VAPlusVetsModal.isEnabled(),
+      showEverytime: true,
+    },
```

Comment:
I moved this to the top of the list because it has highest priority. It does more than a typical Announcement - instead of just rendering based on the path, it needs to check the referrer - so I leveraged the `disabled` property to have it either render or seamlessly disable itself.

Reply:
Other announcement components I think return null if they shouldn't be rendered. Does that not work here? Or is this a better pattern?

Reply:
Announcements were intended to be really simple, and just to be configured for a certain path. If it returns null in the render function, it'll eclipse the other announcements - https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/site-wide/announcements/selectors.js#L13. Maybe it would be better to spruce up how Announcements work in general? This feels like a temporary edge case, but I'm not sure.